packages/calcite-components/src/components/swatch-group/swatch-group.stories.ts

```diff
@@ -0,0 +1,50 @@
+import { modesDarkDefault } from "../../../.storybook/utils";
+import { html } from "../../../support/formatting";
+import { ATTRIBUTES } from "../../../.storybook/resources";
+import { SwatchGroup } from "./swatch-group";
+
+const { selectionMode, scale } = ATTRIBUTES;
+
+type SwatchGroupStoryArgs = Pick<SwatchGroup, "selectionMode" | "scale">;
+
+export default {
+  title: "Components/Swatch Group",
+  args: {
+    selectionMode: selectionMode.defaultValue,
+    scale: scale.defaultValue,
+  },
+  argTypes: {
+    selectionMode: {
+      options: selectionMode.values.filter(
+        (option) => option !== "children" && option !== "multichildren" && option !== "ancestors",
+      ),
+      control: { type: "select" },
+    },
+    scale: {
+      options: scale.values,
+      control: { type: "select" },
+    },
+  },
+};
+
+export const simple = (args: SwatchGroupStoryArgs): string => html`
+  <calcite-swatch-group selection-mode="${args.selectionMode}" scale="${args.scale}">
+    <calcite-swatch></calcite-swatch>
```

Comment:
Should we define some colors for these swatches in the stories, or show a mix of use cases?

Reply:
Updated, though I need to update the image paths.